index.html

```diff
@@ -322,5 +322,5 @@ <h2 id="current_visitors" class="data">...</h2>
   <script src="/js/index.js"></script>
 
   <!-- report to oneself -->
-  <script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA"></script>
+  <script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA&exts=json"></script>
```

Comment:
This is a syntax error, there should be an `&` before `ext=json`, not a space.

Can you fix that, test the event locally to make sure it's firing as expected, and then ping the PR to say it's working?